# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/moz.build
# Commit: 8b26219a4294
# Full Hash: 8b26219a4294a083f2d22da23bd8958234e4dd1b
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-08-16 16:01:29
# Regressor Bug: 1740530
# File Overlap Count: 1
# Description:
#   Bug 1740530 - patch 1 - Move COLR font support functions from gfxUtils into their own class. r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152036
# ==============================================================================

diff -r 8d99a5abccc7 -r 8b26219a4294 gfx/thebes/moz.build
--- a/gfx/thebes/moz.build	Tue Aug 16 06:00:43 2022 -0400
+++ b/gfx/thebes/moz.build	Tue Aug 16 10:39:50 2022 +0000
@@ -17,6 +17,7 @@
 XPIDL_MODULE = "gfx"
 
 EXPORTS += [
+    "COLRFonts.h",
     "DrawMode.h",
     "gfx2DGlue.h",
     "gfxAlphaRecovery.h",
@@ -195,6 +196,7 @@
 
 UNIFIED_SOURCES += [
     "CJKCompatSVS.cpp",
+    "COLRFonts.cpp",
     "gfxAlphaRecovery.cpp",
     "gfxBaseSharedMemorySurface.cpp",
     "gfxBlur.cpp",
